cdap-proto/src/main/java/co/cask/cdap/proto/security/Action.java

```diff
@@ -15,11 +15,14 @@
  */
 package co.cask.cdap.proto.security;
 
+import co.cask.cdap.api.annotation.Beta;
+
 /**
  * Types of actions that users can perform on entities.
  * Actions are inherited, so granting an action on a namespace
  * would also grant that action on entities in that namespace.
  */
+@Beta
 public enum Action {
```

Comment:
We should revisit this and consider keeping consistent Actions in CDAP and other plugin